<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight">
      <number>60</number>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>35</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>58</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass Scaling:</string>
      </property>
     </widget>
     <widget class="ctkPathLineEdit" name="meshFile">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>26</y>
        <width>401</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="ctkTreeComboBox" name="massscaling">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>55</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>true</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>false</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Integration Options</string>
     </property>
     <property name="collapsedHeight">
      <number>185</number>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Adaptive:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>54</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adaptive Eps:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adaptive Level:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tetrahedron Divisions:</string>
      </property>
     </widget>
     <widget class="ctkTreeComboBox" name="adaptive">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>26</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>false</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>true</string>
       </property>
      </item>
     </widget>
     <widget class="ctkDoubleSpinBox" name="adaptiveEps">
      <property name="geometry">
       <rect>
        <x>237</x>
        <y>50</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="value">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="adaptiveLevel">
      <property name="geometry">
       <rect>
        <x>238</x>
        <y>72</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="tDevision">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>100</y>
        <width>60</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>4.000000000000000</double>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="iTetra">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>127</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>4.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>132</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Integration Points Per Tetrahedron:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>For Generating Integration Points File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>183</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Save to File:</string>
      </property>
     </widget>
     <widget class="ctkPathLineEdit" name="intFile">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>181</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="filters">
       <set>ctkPathLineEdit::AllEntries|ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::Executable|ctkPathLineEdit::Files|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::PermissionMask|ctkPathLineEdit::Readable|ctkPathLineEdit::Writable</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Material</string>
     </property>
     <property name="collapsedHeight">
      <number>30</number>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Read From File:</string>
      </property>
     </widget>
     <widget class="ctkPathLineEdit" name="mFile">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>27</y>
        <width>360</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Shape Function</string>
     </property>
     <property name="collapsedHeight">
      <number>111</number>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>34</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>58</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Basic Function Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>83</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Ecact Derivatives: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>107</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dilation Coefficient: </string>
      </property>
     </widget>
     <widget class="ctkTreeComboBox" name="type">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>28</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>mmls</string>
       </property>
      </item>
     </widget>
     <widget class="ctkTreeComboBox" name="bFunction">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>53</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>quadratic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>linear</string>
       </property>
      </item>
     </widget>
     <widget class="ctkTreeComboBox" name="uDerivative">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>78</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>true</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>false</string>
       </property>
      </item>
     </widget>
     <widget class="ctkDoubleSpinBox" name="dilationCoefficient">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>102</y>
        <width>66</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>1.700000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Loading</string>
     </property>
     <property name="collapsedHeight">
      <number>60</number>
     </property>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Read From File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File Load Curve:</string>
      </property>
     </widget>
     <widget class="ctkPathLineEdit" name="lFile">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>26</y>
        <width>351</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="ctkTreeComboBox" name="loadCurve">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>51</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>smooth</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Contacts</string>
     </property>
     <property name="collapsedHeight">
      <number>60</number>
     </property>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Set:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>57</y>
        <width>59</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Surface:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contact">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>27</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Node set defines all the nodes of the brain model in contact with the skull surface&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>contact</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="skull">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>54</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It define the skull elements S3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>skull</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dynamic Relaxation</string>
     </property>
     <property name="collapsedHeight">
      <number>80</number>
     </property>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Time for Generating Integration Points:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>56</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Time for Running Simulation:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Equilibrium Time: </string>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="loadIntegration">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>25</y>
        <width>66</width>
        <height>24</height>
       </rect>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>2.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.001000000000000</double>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="loadRunning">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>66</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time for loading application.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="ctkDoubleSpinBox" name="eTime">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>76</y>
        <width>66</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time for solution to reach equilibrium in dynamic relaxation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Output FIles</string>
     </property>
     <property name="collapsedHeight">
      <number>75</number>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Simulator Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="sPath"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Run Simulator</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkTreeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
